### ISSUE TITLE
Add a license file

### ISSUE THREAD

**Issue by dpsanders** [msylau/measles_competing_risks#1]
Could you please add an open-source license, preferably the [MIT license](https://opensource.org/licenses/MIT), so that people can legally look at and use your code?

**Comment by bbolker** [comment]
Yes please! It might be nice if there were a separate license for the data (more permissive/more relevant to data instead of code).
